Prace_domowe/Praca_domowa2/Jakub_Foltyn/PD2.ipynb

```diff
@@ -0,0 +1,1178 @@
+{
```

Comment:
<p>Doceniam, że wyliczono target encoding samemu, a nie skorzystano z gotowych funkcji :)</p><p> </p><p> </p><p>Zgodnie ze sztuką należałoby najpierw dokonać chociaż minimalnej normalizacji danych typu string - na przykład poprzez zamianę wszystkich znaków na małe litery. Obecnie może się zdarzyć, że stolica będzie zapisana jako Warszawa, WARSZAWA, warszawa i wszystkie te wartości będą traktowane jako inne nazwy innych miast, tym samym na późniejszym etapie będą miały inny encoding.</p><p>Kwestia była poruszana na zajęciach dlatego musiałabym uciąć 0.5 pkt, ale ponieważ ma Pan świadomość tej kwestii (co wykazał Pan w komentarzu poniżej) to wyjątkowo odstąpię od odejmowania punktów w tym aspekcie.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>